### ISSUE TITLE
IsEmailVerified middleware shouldn't throw an exception

### ISSUE THREAD

**Issue by SelimSalihovic** [meness/verifi#6]
Currently it throws a UserNotVerifiedException in case the user's email is not verified. This should be configurable in the config, maybe something like:
```php
//the route we redirect to in case the user is not verified
'middleware_redirect_route' => 'route_name',

//the error message we set in the aforementioned case
'middleware_redirect_error' => 'You haven\'t verified your account yet'
```

Otherwise developers will have to handle this exception in the exception Handler (`app/Exceptions/Handler.php`) or write their custom middleware, which sucks.

Looking for any other thoughts/suggestions before I start working on this. @meness

**Comment by meness** [comment]
You're right about not throwing exceptions. The configurable option seems good. Thanks for your contribution, my friend 👍 

I'm waiting to see how you would handle this.

**Comment by ankurk91** [comment]
@meness 
What about something like this

https://github.com/meness/laravel-email-verification/blob/master/resources/lang/en/messages.php

**Comment by meness** [comment]
@ankurk91 Language file is a better option for messages.

**Comment by karmendra** [comment]
Any PR submitted for this?

**Comment by SelimSalihovic** [comment]
No @karmendra . Feel free to submit one though.
